_maps/map_files/Delta/delta.dmm

```diff
@@ -50607,7 +50608,9 @@
 	},
 /area/station/medical/medbay)
 "cKP" = (
-/obj/structure/chair/office,
+/obj/structure/chair/office{
```

Comment:
These types shouldn't actually be used. Instead of making them anchored, change them to the `/obj/structure/chair` typepath please

Reply:
fixed